### PR TITLE
Adding new policy EnableAllOcsiClients

### PR DESCRIPTION
**Pull request by cartergreen**
This policy works for macOS and has been left out of the documentation for some reason.  This is me doing due diligence and updating our documentation to have all our policies.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/deploy-and-configure-on-macos.md [hunk 1]
```diff
@@ -185,6 +186,22 @@ The example for this setting in the .plist file is:
 <br/> \<key\>DownloadBandwidthLimited\</key\>  <br/> \<int\>(Download Throughput Rate in KB/sec)\</int\>  <br/>
 
 
+### EnableAllOcsiClients
+<a name="EnableAllOcsiClients"> </a>
+
+This setting lets multiple users use the Microsoft 365 Apps for enterprise, Office 2019, or Office 2016 desktop apps to simultaneously edit an Office file stored in OneDrive. It also lets users share files from the Office desktop apps.
+
+> [!IMPORTANT]
+> We recommend keeping this setting enabled to make syncing faster and reduce network bandwidth. [See all our recommendations for configuring the sync app](ideal-state-configuration.md)
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:195]:
```suggestion
> We recommend keeping this setting enabled to make syncing faster and reduce network bandwidth. [See all our recommendations for configuring the sync app](ideal-state-configuration.md).
```

File: OneDrive/deploy-and-configure-on-macos.md [hunk 2]
```diff
@@ -185,6 +186,22 @@ The example for this setting in the .plist file is:
...
+
+If you set this setting's value to **True** or don't set this setting, the Office tab appears in OneDrive sync preferences, and Use Office applications to sync Office files that I open is selected, by default.
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:197]:
```suggestion
If you set this setting's value to **True** or don't set this setting, the Office tab appears in OneDrive sync preferences, and Use Office applications to sync Office files that I open is selected, by default.
```

File: OneDrive/deploy-and-configure-on-macos.md [hunk 3]
```diff
@@ -185,6 +186,22 @@ The example for this setting in the .plist file is:
...
+
+If you set this settings to **False**, the Office tab is hidden in the sync app, and coauthoring and in-app sharing for Office files is disabled. The Users can choose how to handle Office files in conflict setting acts as disabled, and when file conflicts occur, both copies of the file are kept. For more info about the settings in the sync app, see [Use Office applications to sync Office files that I open](https://support.office.com/article/8a409b0c-ebe1-4bfa-a08e-998389a9d823)
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:199]:
```suggestion
If you set this setting's value to **False**, the Office tab is hidden in the sync app, and co-authoring and in-app sharing for Office files are disabled. The **User can choose how to handle Office files in conflict** setting acts as disabled, and when file conflicts occur, both copies of the file are kept. For more information about the settings in the sync app, see [Use Office applications to sync Office files that I open](https://support.office.com/article/8a409b0c-ebe1-4bfa-a08e-998389a9d823).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cartergreen, could you please commit my suggested changes if you agree with them? Many thanks.

**Pull request merged**